docker-compose-canton.yml

```diff
@@ -31,7 +31,7 @@ services:
       start_period: 30s
 
   canton:
-    image: digitalasset/canton:${CANTON_VERSION}
+    image: ${CANTON_IMAGE}:${CANTON_VERSION}
```

Comment:
This is the name for the local docker image built by Docker compose, if should defer from the remote one because it's custom, avoid overwrite/mixups

Reply:
Can we rename it as such then? Somehow I started it locally and changed the env variable, but it started canton 0.18.3

Reply:
We could prefix with `daml-observability/canton` for example. I think you need to cleanup all the containers to start your compose with a different/rebuilt image.

Reply:
So it seems that running `docker compose up` will not build any image if it finds the image in the registries. For it to work we need `docker compose up --build`

Reply:
Docker will reuse cache and existing local images/tags. We should probably mention a procedure to rebuild and use fresh/changed local docker images, with:
```
docker compose build --no-cache
```